libs/ledger-live-common/src/families/evm/api/rpc.common.ts

```diff
@@ -139,22 +163,33 @@ export const getGasEstimation = (
  */
 export const getFeesEstimation = (currency: CryptoCurrency): Promise<FeeData> =>
   withApi(currency, async (api) => {
-    const block = await api.getBlock("latest");
-    const supports1559 = Boolean(block.baseFeePerGas);
+    const currencySupports1559 = await (async () => {
+      const block = await api.getBlock("latest");
+      return Boolean(block.baseFeePerGas);
+    })();
```

Comment:
At first I changed it because I implemented an exeception list as I thought some chains where returning a `baseFeePerGas` while not really supporting EIP1559, realized the bug was coming from somewhere else, removed the exception list but didn't remove the new verification "look".

I can definitely rollback to the previous version if you think it's confusing this way 👌

Reply:
Just personal opinion: it's easier to read in the previous version

Reply:
Alright, then I'm rolling back this part 👍